runtime/lua/vim/lsp/log.lua

```diff
@@ -45,6 +45,11 @@ function log.get_filename()
   return logfilename
 end
 
+--- @param s string
+function log._set_filename(s)
```

Comment:
I think this is valuable enough to not underscore it.

Reply:
Perhaps, however I think this module needs an overhaul anyway.